libs/s25main/network/GameClient.cpp

```diff
@@ -1577,18 +1572,20 @@ void GameClient::SkipGF(unsigned gf, GameWorldView& gwv)
     SetPause(true);
 }
 
-void GameClient::SystemChat(const std::string& text, unsigned char player)
+void GameClient::SystemChat(const std::string& text)
 {
-    if(!ci)
-        return;
-    if(player == 0xFF)
-        player = mainPlayer.playerId;
-    ci->CI_Chat(player, CD_SYSTEM, text);
+    SystemChat(text, mainPlayer.playerId);
```

Comment:
whats the difference between `mainPlayer.playerId` and `GetPlayerId()`?

Reply:
Nothing: `unsigned GetPlayerId() const override { return mainPlayer.playerId; }`

Reply:
then we should probably use `GetPlayerId()` there too, so its clear which one is used

Reply:
Done: 7d27ec83c